### PR TITLE
chore: Guide Enhancements

### PR DESCRIPTION
**Pull request by BruceCodesGithub**
This is a pull request to prevent conflicts.

### CODE REVIEW

**Review by EmreTech — APPROVED** [review]

File: docs/getting-started/more-features.mdx [hunk 1]
```diff
@@ -2,25 +2,30 @@
...
+
+import {
+  DiscordInteraction,
+  DiscordMessage,
+  DiscordMessages,
+  DiscordEmbed,
+  DiscordEmbedField,
+  DiscordEmbedFields,
+} from "@discord-message-components/react";
+import "@discord-message-components/react/styles";
```
Comment (BobDotCom) [inline docs/getting-started/more-features.mdx:14]: Update for the new package name
```suggestion
} from "discord-message-components/packages/react";
import "discord-message-components/packages/react/dist/style.css";
```

File: docs/getting-started/more-features.mdx [hunk 2]
```diff
@@ -68,70 +73,140 @@ Here, we check if the message is from the user that sent the command. We simply
...
+                    <a href="/category/popular-topics">Other Popular Topics</a>
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="And More!">
+                    We have so much more! Just explore, and you will find everything you need. If you want another page added, open an issue on the <a href="https://github.com/Pycord-Development/guide">GitHub</a>.
+                </DiscordEmbedField>
+            </DiscordEmbedFields>
+            <span slot="footer">Created with 💖 by the Pycord Team & Contributers</span>
+        </DiscordEmbed>
+    </DiscordMessage>
+</DiscordMessages>
```
Comment (BobDotCom) [inline docs/getting-started/more-features.mdx:111]:
```suggestion
<DiscordComponent>
    <DiscordMessage author="Guide Bot" avatar="red" bot>
        <DiscordEmbed
        embedTitle="Pycord Guide"
        url="https://guide.pycord.dev"
        authorIcon="/img/logo.png"
        authorName="Pycord Team"
        authorUrl="https://github.com/Pycord-Development/guide"
        thumbnail="/img/logo.png"
        borderColor="#5865F2"
        timestamp="01/09/2021"
        footerIcon="/img/logo.png"
        image="/img/banner-v3.png"
        >
            Pycord Guide is a detailed guide that explains how to use Pycord and all it's features.
            <DiscordEmbedFields slot="fields">
                <DiscordEmbedField fieldTitle="Getting Started">
                    The <a href="/category/getting-started">Getting Started section</a> explains how you can get a brand new bot created and running from scratch.
                </DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Interactions" inline="true">
                    <a href="/interactions">Interactions</a> with Pycord
                </DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Extensions" inline="true">
                    Pycord's various <a href="/category/extensions">Extensions</a>
                </DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Popular Topics" inline="true">
                    <a href="/category/popular-topics">Other Popular Topics</a>
                </DiscordEmbedField>
                <DiscordEmbedField fieldTitle="And More!">
                    We have so much more! Just explore, and you will find everything you need. If you want another page added, open an issue on the <a href="https://github.com/Pycord-Development/guide">GitHub</a>.
                </DiscordEmbedField>
            </DiscordEmbedFields>
            <span slot="footer">Created with 💖 by the Pycord Team & Contributers</span>
        </DiscordEmbed>
    </DiscordMessage>
</DiscordComponent>
```

File: docs/getting-started/more-features.mdx [hunk 3]
```diff
@@ -68,70 +73,140 @@ Here, we check if the message is from the user that sent the command. We simply
...
+                    A really nice field with some information. <strong>The description as well as the fields support markdown!</strong>
+                </DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Inline Field 1" inline="true">Inline Field 1</DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Inline Field 2" inline="true">Inline Field 2</DiscordEmbedField>
+                <DiscordEmbedField fieldTitle="Inline Field 3" inline="true">Inline Field 3</DiscordEmbedField>
+            </DiscordEmbedFields>
+            <span slot="footer">Footer! No markdown here.</span>
+        </DiscordEmbed>
+    </DiscordMessage>
+</DiscordMessages>
```
Comment (BobDotCom) [inline docs/getting-started/more-features.mdx:172]:
```suggestion
<DiscordComponent>
    <DiscordMessage author="Guide Bot" avatar="red" bot>
        <div slot="interactions">
            <DiscordInteraction author="Guide Man" avatar="green" command>hello</DiscordInteraction>
        </div>
        <DiscordEmbed
        embedTitle="My Amazing Embed"
        authorIcon="/img/logo.png"
        authorName="Pycord Team"
        authorUrl="https://github.com/Pycord-Development/guide"
        thumbnail="/img/logo.png"
        borderColor="#5865F2"
        footerIcon="/img/logo.png"
        image="/img/banner-v3.png"
        >
            Embeds are super easy, barely an inconvinience.
            <DiscordEmbedFields slot="fields">
                <DiscordEmbedField fieldTitle="A Normal Field">
                    A really nice field with some information. <strong>The description as well as the fields support markdown!</strong>
                </DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Inline Field 1" inline="true">Inline Field 1</DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Inline Field 2" inline="true">Inline Field 2</DiscordEmbedField>
                <DiscordEmbedField fieldTitle="Inline Field 3" inline="true">Inline Field 3</DiscordEmbedField>
            </DiscordEmbedFields>
            <span slot="footer">Footer! No markdown here.</span>
        </DiscordEmbed>
    </DiscordMessage>
</DiscordComponent>
```

**Review by BobDotCom — DISMISSED** [review]

**Comment by BobDotCom** [comment]
Also need to update those examples to use the new profiles instead of "guide man"/"guide bot"

File: docs/getting-started/more-features.mdx [hunk 4]
```diff
@@ -68,70 +73,140 @@ Here, we check if the message is from the user that sent the command. We simply
...
+        authorIcon="/img/logo.png"
+        authorName="Pycord Team"
+        authorUrl="https://github.com/Pycord-Development/guide"
+        thumbnail="/img/logo.png"
+        borderColor="#5865F2"
+        timestamp="01/09/2021"
+        footerIcon="/img/logo.png"
+        image="/img/banner-v3.png"
+        >
+            Pycord Guide is a detailed guide that explains how to use Pycord and all it's features.
```
Comment (ConchDev) [inline docs/getting-started/more-features.mdx:90]:
```suggestion
            Pycord Guide is a detailed guide that explains how to use Pycord and all of its features.
```

File: docs/getting-started/more-features.mdx [hunk 5]
```diff
@@ -68,70 +73,140 @@ Here, we check if the message is from the user that sent the command. We simply
...
+        embedTitle="My Amazing Embed"
+        authorIcon="/img/logo.png"
+        authorName="Pycord Team"
+        authorUrl="https://github.com/Pycord-Development/guide"
+        thumbnail="/img/logo.png"
+        borderColor="#5865F2"
+        footerIcon="/img/logo.png"
+        image="/img/banner-v3.png"
+        >
+            Embeds are super easy, barely an inconvinience.
```
Comment (ConchDev) [inline docs/getting-started/more-features.mdx:160]:
```suggestion
            Embeds are super easy, barely an inconvenience.
```

File: docs/getting-started/more-features.mdx [hunk 6]
```diff
@@ -68,70 +73,140 @@ Here, we check if the message is from the user that sent the command. We simply
...
 Let's break it down:
  
 ```py
 embed = discord.Embed(
-    title="My Embed",
-    description="Pycord makes embeds easy to use and create!",
-    color=discord.Colour.blue()
-)
+        title="My Amazing Embed",
+        description="Embeds are super easy, barely an inconvinience.",
```
Comment (ConchDev) [inline docs/getting-started/more-features.mdx:182]:
```suggestion
        description="Embeds are super easy, barely an inconvenience.",
```

File: docs/getting-started/more-features.mdx [hunk 7]
```diff
@@ -145,8 +220,7 @@ method, you can set a small image to reside at the top-right of the embed. This
 - Image - With the [`set_image`](https://docs.pycord.dev/en/master/api.html#discord.Embed.set_image)
 method, you can set an image to sit at the bottom of an embed. This has a single `url` kwarg.
  
-There are, of course, more methods and items you can use to configure embeds. Here, we just covered
-the basics. 
+There are a lot more methods and attributes you can use to configure embeds. Here, we just covered the basics. Also remember that all of these values are not nessesary in an embed. An embed may only contain a few of these, for example, only a description, or a title and a description, and so on.
```
Comment (ConchDev) [inline docs/getting-started/more-features.mdx:223]:
```suggestion
There are a lot more methods and attributes you can use to configure embeds. Here, we just covered the basics. Also remember that all of these values are not necessary in an embed. An embed may only contain a few of these, for example, only a description, or a title and a description, and so on.
```

**Review by ConchDev — DISMISSED** [review]
Fix the grammar errors I mentioned in a few comments. "Inconvenience" is spelled with an e, not an i.

**Pull request merged**